sys/net/gnrc/link_layer/lorawan/include/gnrc_lorawan_internal.h

```diff
@@ -161,11 +168,16 @@ typedef struct {
  */
 typedef struct {
     uint8_t activation;     /**< Activation mechanism of the MAC layer */
+    bool adr;               /**< adr bit status */
+    bool adr_ack;           /**< adr ack bit status */
```

Comment:
as stated before, we already have the ADR counters. Therefore we don't need to store this in RAM